src/views/HardwareInstallation.vue

```diff
@@ -380,9 +395,99 @@ export default {
 			showComponentRemoveModal: false,
 			modalComponent: undefined,
 			modalComponentName: "",
+			hasData: false,
+			selectManufacturer: undefined,
 		};
 	},
 	computed: {
+		isLoading() {
+			return this.$store.state.mqtt[
+				"openWB/system/configurable/devices_components"
+			];
+		},
+		getDeviceListGeneric() {
```

Comment:
Computed Properties sind analog zu anderen Eigenschaften zu benennen. `get....` deutet auf eine Methode hin und sollte hier nicht verwendet werden.
```suggestion
		deviceListGeneric() {
```
Das gilt auch für die anderen Methoden unter "computed".

Reply:
Ok. Danke. Wird angepasst